india_compliance/gst_india/api_classes/nic/e_waybill.py

```
@@ -0,0 +1,206 @@
+        self.set_default_headers()
+
+    @change_base_path("standard/ewb")
+    def authenticate(self):
+        json_data = {
+            "action": "ACCESSTOKEN",
+            self.USER_NAME: self.username,
+            self.PASSWORD: self.password,
+            self.APP_KEY: self.app_key,
+        }
+
+        return self.post(endpoint="auth", json=json_data)
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'endpoint' in method call

(E1123)